src/pim-types.ts

```diff
@@ -80,6 +80,10 @@ export class EventType extends ICAL.Event implements PimType {
     return this.summary;
   }
 
+  set title(_title: string) {
```

Comment:
The underscore prefix convention is for unused parameters, so shouldn't be used here.

Note: I know it's used elsewhere in the code for something else, but still, nothing to be copied, and was never just used like this.